client/src/app/site/assignments/components/assignment-detail/assignment-detail.component.html

```diff
@@ -253,24 +251,14 @@ <h4>{{ 'Description' | translate }}:</h4>
 
             <!-- Attachments -->
             <div class="content-field" *ngIf="mediafilesAvailable">
```

Comment:
remove the `*ngIf` and all mediafile related code (e.g. the property and the `mediafilesObserver` member) from the component.